### ISSUE TITLE
Icons missing / invisible inside the windows 

### ISSUE THREAD

**Issue by Rishishukla6** [snwh/paper-gtk-theme#89]
This is how window appears when opened.  
![screenshot from 2015-02-28 00 45 15](https://cloud.githubusercontent.com/assets/10980039/6419472/7b4f29c4-bee3-11e4-85b6-45dcbece78ca.png)

the options which should be visible by default in side the window appeaser only after moving cursor near to it as shown.

![screenshot from 2015-02-28 00 45 10](https://cloud.githubusercontent.com/assets/10980039/6419500/ca703188-bee3-11e4-91c7-ad9a7f15771d.png)

![screenshot from 2015-02-28 00 50 56](https://cloud.githubusercontent.com/assets/10980039/6419513/e713b954-bee3-11e4-8a0c-752acae78a96.png)

![screenshot from 2015-02-28 00 51 01](https://cloud.githubusercontent.com/assets/10980039/6419514/e7241416-bee3-11e4-8dc8-e78ffa83a453.png)

**Comment by snwh** [comment]
Duplicate of #58
